test/rubygems/test_gem_resolver_api_specification.rb

```diff
@@ -31,13 +31,14 @@ def test_initialize
   def test_fetch_development_dependencies
     specs = spec_fetcher do |fetcher|
       fetcher.spec 'rails', '3.0.3' do |s|
+        s.platform = 'i386-mingw32'
```

Comment:
Should we have another test which modifies this so we don't change coverage of the original tests? i.e. add a separate test that does this